### ISSUE TITLE
@types/bitcoinjs-lib: `ECPair` should expose `compressed`

### ISSUE THREAD

**Issue by vsund** [DefinitelyTyped/DefinitelyTyped#24453]
Hey @mhegazy @rbuckton @micksatana @youssefgh,

I think that [`ECPair`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9768aa9695b70722b21fbb7098bd4aec205b103b/types/bitcoinjs-lib/index.d.ts#L67) should expose an attribute `readonly compressed: boolean`.

It's defined in `bitcoinjs-lib` [`src/ecpair.js#L37`](https://github.com/bitcoinjs/bitcoinjs-lib/blob/91ff39d95a2ea15def83b1dee98e8aaffba433f7/src/ecpair.js#L37). (There are two more attributes that could get exposed, but I'm not sure whether they should.)

I'm currently porting a library that accesses `compressed` manually. Happy to provide a PR, if you confirm this.

**Comment by youssefgh** [comment]
IMO 
```js
ECPair.prototype.getCompressed = function () {
  return this.compressed
}
```
Should be added first to bitcoinjs-lib, like what do we have for getNetwork().
That would also help others that might need access to compressed (or other properties), but don't use Typescript

**Comment by vsund** [comment]
Yeah, saw that method on some other types too while researching what to to with `compressed`. I'm going to open a PR on `bitcoinjs-lib` tomorrow.

**Comment by vsund** [comment]
For reference, just raised a PR against `bitcoinjs-lib`: https://github.com/bitcoinjs/bitcoinjs-lib/pull/1025

**Comment by vsund** [comment]
As per https://github.com/bitcoinjs/bitcoinjs-lib/pull/1025 we're concluded with adding read-only attributes for `compressed` and `network`.

@youssefgh Do you like to take this?

cc @dcousens

**Comment by youssefgh** [comment]
Yes @vsund, you can make a PR
Thank you

**Comment by vsund** [comment]
Should I add tests for such a simple thing? @youssefgh

**Comment by youssefgh** [comment]
No this is only a typing project with no business logic.
You don't need to
